ui/ui-frontend/projects/vitamui-library/src/sass/_animations.scss

```
@@ -0,0 +1,113 @@
+}
+
+// Fade in/out (menu opacityAnimation)
+.fade-in {
+  animation: menuFadeIn 500ms cubic-bezier(0, 0, 0.2, 1);
+}
+
+.fade-out {
+  animation: menuFadeOut 500ms cubic-bezier(0, 0, 0.2, 1) forwards;
+}
+
+@keyframes menuFadeIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "menuFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)